src/sentry/consumers/__init__.py

```diff
@@ -266,7 +267,7 @@ def ingest_events_options() -> list[click.Option]:
         "strategy_factory": "sentry.ingest.consumer.factory.IngestStrategyFactory",
         "click_options": ingest_events_options(),
         "static_args": {
-            "consumer_type": "feedback-events",
+            "consumer_type": IngestConsumerType.Feedback,
```

Comment:
While we're here let's go ahead and implement this for events, transactions, and attachments as well.